homeassistant/components/met_lightning/config_flow.py

```diff
@@ -0,0 +1,61 @@
+"""Config flow to configure the Met lightning integration."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.helpers import config_validation as cv
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE, CONF_RADIUS
+from homeassistant.core import callback
+
+from .const import DEFAULT_RADIUS, DOMAIN
+
+
+@callback
+def configured_instances(hass):
+    """Return a set of configured Met lightning instances."""
+    return set(
+        "{0}, {1}".format(entry.data[CONF_LATITUDE], entry.data[CONF_LONGITUDE])
+        for entry in hass.config_entries.async_entries(DOMAIN)
+    )
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class MetLightningFlowHandler(config_entries.ConfigFlow):
```

Comment:
Pass the domain as the domain keyword argument and remove the decorator.
https://developers.home-assistant.io/docs/en/next/config_entries_config_flow_handler.html#defining-your-config-flow